### PR TITLE
[13.0][ADD] Pricelist brand

### PR DESCRIPTION
**Pull request by qgroulard**
This module allows to configure pricelist items so that they are applied on products of specific brands.

image

### CODE REVIEW

File: pricelist_brand/models/product_pricelist.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+    def _compute_price_rule_get_items(
+        self, products_qty_partner, date, uom_id, prod_tmpl_ids, prod_ids, categ_ids
+    ):
+        self.ensure_one()
+        product_tmpls = self.env["product.template"].browse(prod_tmpl_ids)
+        brand_ids = product_tmpls.mapped("product_brand_id").ids
+        # Load all rules
+        self.env["product.pricelist.item"].flush(["price", "currency_id", "company_id"])
+        self.env.cr.execute(
```
Comment (sbejaoui) [inline pricelist_brand/models/product_pricelist.py:20]: Unfortunately this query make it impossible to inherit this method correctly from the base module. I would suggest a PR to odoo to refactor this query in different methods so we can add a new pricelist option without overriding the standard code.

**Review by sbejaoui — APPROVED** [review]
I approve this PR even with the inheritance issue in `_compute_price_rule_get_items` method as we can't do better without a refactoring of it in the base module.

In the meantime I suggest to add a note in the readme as known issue to alarm about the risk of installing this module if another one did th same override of `_compute_price_rule_get_items`

**Comment by elvise** [comment]
Hi @sbejaoui thanks for your PR! 
Question: What happens if i have to set up "Brand 1" and "Shoe" category?

**Comment by elvise** [comment]
@qgroulard any chance for merge it ?

**Pull request merged**